src/components/header/header.tsx

```diff
@@ -0,0 +1,39 @@
+type HeaderProps = {
+  isNavigation: boolean;
```

Comment:
Nit: лучше читается кмк
```suggestion
  hasNavigation: boolean;
```